torch/csrc/jit/passes/quantization.cpp

```diff
@@ -697,6 +698,41 @@ graph(%a_dequant, %w_quant, %b, %stride, %padding, %dilation, %groups):
   rewriter.runOnGraph(graph);
 }
 
+void insertPrepackUnpackForConvolution(std::shared_ptr<Graph>& graph) {
```

Comment:
please change this to replacing `aten::_convolution` to `aten::conv2d` as discussed.